shotover-proxy/examples/redis-cluster-ports-rewrite/topology.yaml

```diff
@@ -2,13 +2,13 @@
 sources:
   redis_prod:
     Redis:
-      listen_addr: "127.0.0.1:6379"
+      listen_addr: "0.0.0.0:6380"
 chain_config:
   redis_chain:
     - RedisClusterPortsRewrite:
-         new_port: 2004
+         new_port: 6380
     - RedisSinkCluster:
```

Comment:
Should this be a cluster or a single?

Reply:
Both can be used

Reply:
I can switch to RedisSinkSingle though

Reply:
:+1: 

I dont think they are interchangeable, I think RedisSinkCluster would try to connect to other instances if it received a MOVE?
But I think RedisSinkSingle is what we want as its a straight pass through.